tests/TestCase/src/Controller/ManufacturersFrontendControllerTest.php

```diff
@@ -31,42 +33,41 @@ public function setUp(): void
 
        public function testManufacturerDetailOnlinePublicLoggedOut()
     {
-        $this->httpClient->followOneRedirectForNextRequest();
-        $this->httpClient->get($this->Slug->getManufacturerDetail(4, 'Demo Manufacturer'));
-        $this->assert200OkHeader();
+        $this->get($this->Slug->getManufacturerDetail(4, 'Demo Manufacturer'));
```

Comment:
To avoid the additional get-request in line 37, assure that the first one is not redirected by passing the correct manufacturer name:
$this->get($this->Slug->getManufacturerDetail(4, 'Demo Fleisch Hersteller'));
Then you should be able to drop line 37.

Reply:
At first I wasn't sure if I can do this as most of them had `Demo Manufacturer` and I thought that it was mandatory. But, now I understand. Thank you for your suggestion.